torch/_prims/__init__.py

```diff
@@ -1094,13 +1116,7 @@ def _mul_nvfuser(fd: Any, a: TensorLikeType, b: TensorLikeType):
 pow = _make_elementwise_binary_prim(
     "pow",
     impl_aten=torch.pow,
-    doc="",
-    type_promotion=ELEMENTWISE_PRIM_TYPE_PROMOTION_KIND.DEFAULT,
-)
-
-rsqrt = _make_elementwise_binary_prim(
```

Comment:
`rsqrt` is not a binary prim, but unary. How can we prevent these mistakes in the future?

Reply:
We just need to add a reference for it so it's tested (currently we're only testing refs directly and prims indirectly through them)

Reply:
We just need to add a reference for it so it's tested (currently we're only testing refs directly and prims indirectly through them)